app/TipsDetails.tsx

```diff
@@ -0,0 +1,115 @@
+import { Dimensions, ScrollView, StyleSheet, Text, View } from 'react-native';
+
+import { TIPS_INFO } from './data/TipsInfo';
+import LightbulbIcon from './icons/LightbulbIcon';
+
+const screenHeight = Dimensions.get('window').height;
+const screenWidth = Dimensions.get('window').width;
+
+const TipsHeader = ({ schoolName }: { schoolName: string }) => (
+  <View style={style.headerContainer}>
+    <Text style={style.header}>{schoolName}</Text>
+  </View>
+);
+
+const TipsDetails = ({ schoolName }: { schoolName: string }) => {
+  const tipArray = []; //create new array to keep track of which tips are from the school
+
+  console.log(schoolName);
+
+  // loops over TIPS_INFO object and pulls which key:value pairs coordinate to the relevant school
+  for (const key in TIPS_INFO) {
+    const value = TIPS_INFO[key];
+    if (value.site === schoolName) {
+      tipArray.push(value);
+    }
+  }
+
+  // Rendering the header and a list of tips
+
+  return (
+    <ScrollView contentContainerStyle={style.scrollViewContentContainer}>
+      <Text style={style.title}>Tips!</Text>
+      <TipsHeader schoolName={schoolName} />
+      <View style={style.section}>
+        {tipArray.map((tip) => (
+          <View style={style.standoutText}>
+            <LightbulbIcon />
+            <Text style={style.text}>{tip.content} </Text>
+          </View>
+        ))}
+      </View>
+    </ScrollView>
+  );
+};
+
+export default TipsDetails;
+
+const style = StyleSheet.create({
```

Comment:
Can we make the "Tips!" title smaller, left align the text, make the icon a little smaller just to match the Figma mockup?

Reply:
Or some texts are displayed a bit weirdly with tabs/spaces